mmdet/datasets/custom.py

```diff
@@ -72,24 +73,31 @@ def __init__(self,
         self.test_mode = test_mode
         self.filter_empty_gt = filter_empty_gt
         self.CLASSES = self.get_classes(classes)
+        self.file_client = mmcv.FileClient(**file_client_args)
```

Comment:
We do not need file_client now. If we use the newest mmcv, we can directly use mmcv.load() to load pkl files on ceph

Reply:
We need to map the file path from local to ceph, this requires file_client.